### DISCUSSION TITLE
How do i access all tools for a plugin?

### DISCUSSION THREAD

**Discussion by Doomswan** [xournalpp/xournalpp#4236] [Q&A] [answered]
I've been trying to build a plugin but I'm new to this so i don't really know where i can find the specific references to the tools. any help will be greatly appreciated.

**Comment by rolandlo** [comment, marked as answer]
Have you seen the guide on the website, in particular [this section](https://xournalpp.github.io/guide/plugins/plugins/#using-plugins-to-define-shortcuts-for-ui-actions)?

**Reply by Doomswan** [reply]
Thanks. I did not titice it at all
